packages/modules/src/modules/banners/choiceCardsButtonsBanner/components/ChoiceCardFrequencyTabs.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import {
+    ContributionAmounts,
+    ContributionFrequency,
+    OphanComponentEvent,
+} from '@sdc/shared/dist/types';
+import { ChoiceCardBannerComponentId } from './ChoiceCards';
+import { ChoiceCardSelection } from '../ChoiceCardsButtonsBanner';
+import { ChoiceCard } from '@guardian/src-choice-card';
+import { css } from '@emotion/react';
+import { space } from '@guardian/src-foundations';
+
+const container = css`
+    display: flex;
+
+    > label {
+        margin-right: ${space[2]}px !important;
+        margin-bottom: ${space[3]}px !important;
+    }
+
+    > label:last-of-type {
+        margin-right: 0 !important;
+    }
+`;
+
+interface ContributionTypeItem {
+    label: string;
+    frequency: ContributionFrequency;
+    suffix: string;
+}
+
+export type ContributionType = {
+    [key in ContributionFrequency]: ContributionTypeItem;
+};
+
+export const trackClick = (
+    type: 'amount' | 'frequency',
+    componentId: ChoiceCardBannerComponentId,
+    submitComponentEvent?: (event: OphanComponentEvent) => void,
+): void => {
+    if (submitComponentEvent) {
+        submitComponentEvent({
+            component: {
+                componentType: 'ACQUISITIONS_OTHER',
+                id: `${componentId}-change-${type}`,
+            },
+            action: 'CLICK',
+        });
+    }
+};
+
+export const ChoiceCardFrequencyTabs = ({
+    componentId,
+    submitComponentEvent,
+    amounts,
+    setSelectionsCallback,
+    selection,
+}: {
+    componentId: ChoiceCardBannerComponentId;
+    submitComponentEvent?: (event: OphanComponentEvent) => void;
+    amounts: ContributionAmounts;
+    setSelectionsCallback: (choiceCardSelection: ChoiceCardSelection) => void;
+    selection: ChoiceCardSelection;
+}): JSX.Element => {
+    const updateFrequency = (frequency: ContributionFrequency) => {
+        trackClick('frequency', componentId, submitComponentEvent);
+        amounts &&
+            setSelectionsCallback({
+                frequency: frequency,
+                amount: amounts[frequency].defaultAmount,
+            });
+    };
+
+    const tabFrequencies: ContributionFrequency[] = ['ONE_OFF', 'MONTHLY', 'ANNUAL'];
+
+    const getRecurringLabelText = (tabFrequency: ContributionFrequency) =>
+        tabFrequency[0] + tabFrequency.slice(1).toLowerCase();
+
+    const tabList = tabFrequencies.map(tabFrequency => ({
+        frequency: tabFrequency,
+        id: `banner-${tabFrequency}`,
+        labelText: tabFrequency === 'ONE_OFF' ? 'Single' : getRecurringLabelText(tabFrequency),
+        selected: selection.frequency === tabFrequency,
+    }));
+
+    return (
+        <div css={container}>
+            {tabList.map(tab => (
+                <ChoiceCard
```

Comment:
Can see the cards being shown rather than frequency tabs.

Reply:
Worth renaming this component 'ChoiceCardFrequencyCards.tsx'?

Reply:
Started with a copy-paste here as we'll end up with one or the other, so I'll leave other names as is for now